test/system/800-config.bats

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test specific configuration options and overrides
+#
+
+load helpers
+
+@test "podman CONTAINERS_CONF - CONTAINERS_CONF in conmon" {
+    skip_if_remote "can't check conmon environment over remote"
+
+    # Make an innocuous containers.conf in a non-standard location
+    conf_tmp="$PODMAN_TMPDIR/containers.conf"
+    cat >$conf_tmp <<EOF
+[engine]
+runtime="${OCI_RUNTIME:-crun}"
+EOF
+    CONTAINERS_CONF="$conf_tmp" run_podman run -d $IMAGE sleep infinity
+    cid="$output"
+
+    CONTAINERS_CONF="$conf_tmp" run_podman inspect "$cid" --format "{{ .State.ConmonPid }}"
+    conmon="$output"
+
+    output="$(tr '\0' '\n' < /proc/$conmon/environ | grep '^CONTAINERS_CONF=')"
+    is "$output" "CONTAINERS_CONF=$conf_tmp"
+
+    # Clean up
+    # Weirdly, podman stop / SIGTERM doesn't seem to work here
+    CONTAINERS_CONF="$conf_tmp" run_podman rm -f -t 0 "$cid"
```

Comment:
Oh, one more: you can remove the comment (no longer applicable) and the `CONTAINERS_CONF=` (not needed for `rm`)

Reply:
Well.. it's applicable in the sense that that's the reason we have `-f -t 0` to force an immediate SIGKILL.

We'd probably be fine without the `CONTAINERS_CONF` setting in this case, but the container was created with the alternate configuration, so AFAICT it makes sense to use the same configuration when removing it.

Reply:
I also note that the `-t` option doesn't appear to be documented in `man podman-rm` :/

Reply:
>force an immediate SIGKILL.

Oh. Yeah, I forgot that that's weird (I just consider it annoying). You can't interrupt `sleep`. (Wish I could get that patch for myself). OK, leaving the `CONTAINERS_CONF=` is fine, I can see your point.

Re `-t`: it's relatively new, might not have propagated to your install yet. Rest assured we have hooks that guarantee that all non-hidden options are documented in man pages.